### DISCUSSION TITLE
BIC sollte kein Pflichtfeld mehr sein

### DISCUSSION THREAD

**Discussion by tolot27** [openjverein/jverein#335] [General]
Die BIC ist ja für Überweisungen im SEPA-Raum schon lange nicht mehr notwendig. Bei SEPA-Lastschriften ist sie im Inland auch schon länger nicht mehr notwendig, nur noch bei Lastschriften von ausländischen Konten. Daher schlage ich vor, dass dies bei den Mitgliedern und insbesondere beim Import kein Pflichtfeld mehr ist bzw. nur dann Pflicht ist, wenn die Landeskennung der IBAN von der des eigenen Kontos abweicht.
Wenn man die IBAN manuell eingibt, macht er ja einen automatischen Lookup. Das ist schon ganz praktisch. Nur bei importierten Mitgliedern wird es nicht gemacht. Wir erfassen auch gar keine BIC mehr.

**Comment by lenilsas** [comment]
habe ich in #339 umgesetzt
